contracts/test/mocks/kill_switch/FailingKillSwitchMock.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity 0.4.24;
+
+import "../../../kill_switch/KillSwitch.sol";
+
+
+contract FailingKillSwitchMock is KillSwitch {
```

Comment:
Nitpick: perhaps we can reword the name and error to be "revert" for clarity ("failing" is ambiguous as to if it doesn't work correctly by returning bad values or by reverting).